modules/backend/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.23)
+project(desktop-business-app-backend)
+
+set(CMAKE_CXX_STANDARD 20)
+
+#TODO: adjust both when we have final version of structure of our project
+set(EXTERNAL_LIBRARIES_DIR ${PROJECT_SOURCE_DIR}/../../external_libraries/)
```

Comment:
Is  PROJECT_SOURCE_DIR a main level -> '/ '?
If so, then I understand why /../../ are necessary :)

Reply:
PROJECT_SOURCE_DIR if I understood copilot right, is the main cmake level dir (or perhaps where PROJECT_NAME var is established. We will have to modify it when we get into final project structure.
In this example it is backend module dir if I got it right